src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+function Calculator() {
+  return (
+    <div className="calc">
+      <span className="keys4">0</span>
+      <span className="key1">AC</span>
+      <span className="key1">+/-</span>
+      <span className="key1">%</span>
+      <span className="key1b">÷</span>
+      <span className="key1">7</span>
+      <span className="key1">8</span>
+      <span className="key1">9</span>
+      <span className="key1b">*</span>
+      <span className="key1">4</span>
+      <span className="key1">5</span>
+      <span className="key1">6</span>
+      <span className="key1b">-</span>
+      <span className="key1">1</span>
+      <span className="key1">2</span>
+      <span className="key1">3</span>
+      <span className="key1b">+</span>
+      <span className="keys2">0</span>
+      <span className="key1">.</span>
+      <span className="key1">=</span>
+    </div>
```

Comment:
- [ ] Please ensure that the background color of the "=" button is the same as that of the other operators just like in the original design. Please see the annotated diagram below for more information 👇 
### Original Design
![image](https://github.com/jcmunav63/math/assets/46839548/b42aed6e-3b59-4a79-975f-f1d3982e5f12)
### Your Design
![image](https://github.com/jcmunav63/math/assets/46839548/adaa80a6-27b5-4ec3-b59e-5fc696d6121b)